proto/viam/component/board/v1/board.proto

```diff
@@ -117,14 +111,9 @@ service BoardService {
   }
 }
 
-message StatusRequest {
-  string name = 1;
-  // Additional arguments to the method
-  google.protobuf.Struct extra = 99;
-}
-
-message StatusResponse {
-  common.v1.BoardStatus status = 1;
+message Status {
+  map<string, viam.common.v1.AnalogStatus> analogs = 1;
+  map<string, viam.common.v1.DigitalInterruptStatus> digital_interrupts = 2;
```

Comment:
it's part of the scope to remove Analog and Digital statuses, however it is breaking app. I think it's fixable, but it requires changes to the TS sdk and the TS sdk cannot be tested until these api changes are merged. I'd like to start with this non-breaking merge and then update typescript and then come back and make the final api change to remove the analog and digital statuses. thoughts @susmitaSanyal

Reply:
sounds good!